data/json/items/ammo.json

```diff
@@ -391,6 +391,7 @@
     "color": "brown",
     "description": "Feathers from a bird.  Useful for fletching arrows.",
     "material": "cotton",
```

Comment:
Or maybe they should not be made of cotton? Isn't cotton a plant-based material anyway?